dcicutils/s3_utils.py

```diff
@@ -148,6 +158,8 @@ def __init__(self, outfile_bucket=None, sys_bucket=None, raw_file_bucket=None,
                 outfile_bucket = self.OUTFILE_BUCKET_TEMPLATE % env
                 raw_file_bucket = self.RAW_BUCKET_TEMPLATE % env
                 blob_bucket = self.BLOB_BUCKET_TEMPLATE % env
+                metadata_bucket = self.METADATA_BUCKET_TEMPLATE % env
+                tibanna_logs_bucket = self.TIBANNA_LOGS_BUCKET_TEMPLATE % env
```

Comment:
I would leave this as `None` for now, or simply `tibanna-logs`.

Reply:
Well, if that variable value was set correctly, it would work this way.